ui/app/templates/components/job-editor.hbs

```diff
@@ -15,28 +15,26 @@
         <p>
           Paste or author HCL or JSON to submit to your cluster, or select from a list of templates. A plan will be requested before the job is submitted. You can also attach a job spec by uploading a job file or dragging &amp; dropping a file to the editor.
         </p>
-
-        {{#if (can "read variable" path="nomad/job-templates/*" namespace="*")}}
```

Comment:
Psst: review this with "Hide Whitespace" enabled. This change makes it so that you can still upload a jobspec without the "read variables" ACL capability, which I had foolishly wrapped the upload button in before (it being next to the button that actually requires variable read access, "Choose from template")